Formula/qemu.rb

```diff
@@ -22,10 +22,12 @@ class Qemu < Formula
   end
 
   devel do
-    url "https://download.qemu.org/qemu-3.0.0-rc4.tar.xz"
-    sha256 "420409d0c47169c4323b29d8af0a14d8276f4f038a74f37891829650dfb9d8cd"
+    url "https://download.qemu.org/qemu-3.0.0.tar.xz"
+    sha256 "8d7af64fe8bd5ea5c3bdf17131a8b858491bcce1ee3839425a6d91fb821b5713"
   end
 
+  option "with-hvf", "Enable Hypervisor.framework support (3.x)"
```

Comment:
It this is compatible with all the other options we offer in 3.x just turn it on by default, unless there's another good reason not to.